compiler/f8cstatic.hpp

```diff
@@ -64,8 +64,11 @@ const BaseTypeMap FieldSpec::_baseTypeMap
 	{"EXCHANGE", FieldTrait::ft_Exchange},
 	{"MONTHYEAR", FieldTrait::ft_MonthYear},
 	{"UTCTIMESTAMP", FieldTrait::ft_UTCTimestamp},
+	{"UTCTIMESTAMPUS", FieldTrait::ft_UTCTimestampUs},
 	{"UTCTIME", FieldTrait::ft_UTCTimeOnly},
+	{"UTCTIMEUS", FieldTrait::ft_UTCTimeOnlyUs},
 	{"UTCTIMEONLY", FieldTrait::ft_UTCTimeOnly},
+	{"UTCTIMEONLYUS", FieldTrait::ft_UTCTimeOnlyUs },
```

Comment:
This field type is not specified in the FIX specification. As a non-standard extension, schemas that use this will not be compatible with standard types.
I think a better approach is to make this a build option, leaving the schema unchanged. The current spec [already allows for ms,us or ns](http://fixwiki.org/fixwiki/UTCTimeOnlyDataType).